.github/workflows/continuous-integration.yml

```diff
@@ -54,50 +88,67 @@ jobs:
         path: rrtmgp-data
         ref: v1.8.1
     #
-    # Synchronize the package index
-    #
-    - name: Synchronize the package index
-      run: sudo apt-get update
-    #
-    # Install NetCDF-Fortran (compatible with all compilers)
-    #
-    - name: Install NetCDF-Fortran
-      run: sudo apt-get install libnetcdff-dev
-    #
     # Cache Conda packages
     #
     - name: Cache Conda packages
       uses: actions/cache@v4
       with:
         path: ~/conda_pkgs_dir
-        key: conda-pkgs
+        key:
+          ${{ runner.os }}-conda-${{ hashFiles('environment-ci.yml') }}
     #
     # Set up Conda
     #
     - name: Set up Conda
       uses: conda-incubator/setup-miniconda@v3
       with:
         miniforge-version: latest
-        activate-environment: rte_rrtmgp_test
-        environment-file: environment-noplots.yml
-        python-version: 3.11
+        activate-environment: rte_rrtmgp_ci
+        environment-file: environment-ci.yml
         auto-activate-base: false
+        use-mamba: true
         # Use the cache properly:
         use-only-tar-bz2: true
     #
+    # Install NetCDF-Fortran
+    #
+    - name: Install NetCDF-Fortran via Conda
+      if: runner.os == 'Linux'
+      run: conda install -y -c conda-forge netcdf-fortran
+    #
+    # Setup Linux environment variables
+    #
+    - name: Setup env variables - Linux
+      if: runner.os == 'Linux'
+      run: |
+        echo FCINCLUDE=-I ${CONDA_PREFIX}/include >> $GITHUB_ENV
+        echo LDFLAGS=-L ${CONDA_PREFIX}/lib >> $GITHUB_ENV
+        echo LD_LIBRARY_PATH=${CONDA_PREFIX}/lib:${LD_LIBRARY_PATH} >> $GITHUB_ENV
```

Comment:
```suggestion
        echo "FCINCLUDE=-I${CONDA_PREFIX}/include" >> $GITHUB_ENV
        echo "LDFLAGS=-L${CONDA_PREFIX}/lib" >> $GITHUB_ENV
        echo "LD_LIBRARY_PATH=${CONDA_PREFIX}/lib:${LD_LIBRARY_PATH}" >> $GITHUB_ENV
```

Reply:
These are equivalent I believe, just different styling, so happy to change those.